app/routes/public.js

```diff
@@ -26,14 +30,28 @@ module.exports = [{
   }
 }, {
   method: 'GET',
-  path: '/assets/{path*}',
+  path: '/node_modules/govuk-frontend/govuk/{path*}',
```

Comment:
Does the path need to include node_modules? For brevity would `/govuk-frontend/govuk/{path*}` be enough?

Reply:
having node modules in the path is a bad idea, I'll get it changed.